ddtrace/contrib/internal/tornado/patch.py

```diff
@@ -50,10 +50,8 @@ def patch():
     _w("tornado.template", "Template.generate", template.generate)
 
     # configure the global tracer
-    ddtrace.tracer._configure(
-        context_provider=context_provider,
-        wrap_executor=decorators.wrap_executor,
-    )
+    ddtrace.tracer.context_provider = context_provider
```

Comment:
I don't think we need this. These values are set in when the torando app is started (in the diff above) but I want to limit the scope of the refactor to removing `tracer._configure(...)` and only making critical changes.